server/topic.go

```diff
@@ -1242,10 +1258,17 @@ func (t *Topic) thisUserSub(h *Hub, sess *Session, pkt *ClientComMessage, asUid
 			}
 
 			changed = true
```

Comment:
It looks like it's the only place where `changed` is modified.
Otherwise, it's always false. Is it also reasonable to give it a better name, e.g. `newSub`?

Reply:
It's in two places: it's true either for created or updated subscription.